libs/clients/passports/src/lib/passportsApi.service.ts

```diff
@@ -113,12 +113,14 @@ export class PassportsService {
 
   async getIdentityDocument(
     auth: User,
+    type?: string | null,
```

Comment:
Can this be a enum or a literal? It's not apparent what the possible types are, so if I were using this function I'd be forced to guess what the type _could_ be which isn't ideal